### ISSUE TITLE
"Print" option

### ISSUE THREAD

**Issue by richukuttan** [elias-sundqvist/obsidian-annotator#102]
Hi,

It would be really great if the print option in the pdf reader actually printed out a pdf with the annotations. Instead, it currently screenshots each page and compiles to create a really large pdf - with no annotations present.

This is not an urgent feature, but a good-to-have.

Thanks for your work!

**Comment by ryanwwest** [comment]
I agree, this would be nice to have, or some way to export the annotations in pdf/epub.

**Comment by madukan** [comment]
Perhaps it is a bug in the past? I just now tested myself and the print option creates (indeed a very large) PDF with screenshot of the page on the left and annotations on the right. I landed here trying to find a solution for the problem of [annotated texts being cropped during the annotation exporting](https://github.com/elias-sundqvist/obsidian-annotator/issues/389) pdf.
